corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -63,14 +63,15 @@ <h4 class="modal-title">{% trans "Create a new Case Type" %}</h4>
                   <label for="description" class="col-xs-12 col-sm-4 col-md-4 col-lg-2 control-label">
                       {% trans "Description" %}
                   </label>
-                  <div class="col-xs-12 col-sm-8 col-md-8 col-lg-8 controls">
+                  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-8 controls">
                     <textarea name="description" class="form-control"></textarea>
                   </div>
                 </div>
               </fieldset>
             </div>
             <div class="modal-footer">
               <a href="#" data-dismiss="modal" class="btn btn-default" data-bind="
+                click: clearForm,
```

Comment:
This won't run if someone uses escape or clicks in the background to close the modal. If you use an `event` binding on the modal itself with `hide.bs.modal` you can pick up those actions, too.

Reply:
updated